client/docker/images.go

```diff
@@ -29,6 +29,27 @@ import (
 	dockerjsonmessage "github.com/docker/docker/pkg/jsonmessage"
 )
 
+const (
+	sealerDockerString = `sealer`
+	registryAuthString = `sea.hub`
+	registryName       = `sea.hub:5000`
+)
+
+func (d Docker) ImagesCacheToRegistry(images []string) error {
+	version, err := d.cli.ServerVersion(d.ctx)
```

Comment:
whether is sealer-docker or not, we can move out this as an utils flag. And in some scenarios we don’t even have docker, we can try to use cmd "docker version | grep sealer".